### PR TITLE
Allow cloud_init create dhclient var files

### PR DESCRIPTION
**Pull request by 5umm3r15**
Resolves: rhbz#2225418

### CODE REVIEW

**Comment by 5umm3r15** [comment]
Waiting for the test results

**Comment by 5umm3r15** [comment]
Tests passed, ready for the review @zpytela

File: policy/modules/contrib/cloudform.te [hunk 1]
```diff
@@ -164,10 +164,13 @@ optional_policy(`
 ')
 
 optional_policy(`
+    sysnet_domtrans_dhcpc(cloud_init_t)
     sysnet_domtrans_ifconfig(cloud_init_t)
     sysnet_read_dhcpc_state(cloud_init_t)
     sysnet_dns_name_resolve(cloud_init_t)
     sysnet_filetrans_cloud_net_conf(cloud_init_t)
+    sysnet_filetrans_dhcpc_pid(cloud_init_t, "dhclient.pid")
+    sysnet_filetrans_dhcpc_pid(cloud_init_t, "dhclient.lease")
```
Comment (zpytela) [inline policy/modules/contrib/cloudform.te]: With sysnet_domtrans_dhcpc(cloud_init_t) in place, is a file transition still needed?

File: policy/modules/system/init.te [hunk 2]
```diff
@@ -870,6 +870,8 @@ optional_policy(`
 
 optional_policy(`
     sysnet_filetrans_cloud_net_conf(init_t)
+    sysnet_manage_config_dirs(init_t)
```
Comment (zpytela) [inline policy/modules/system/init.te]: Is this really needed?

File: policy/modules/contrib/cloudform.te (hunk 1 above)
Comment (5umm3r15) [reply policy/modules/contrib/cloudform.te]: no, thank you

**Comment by zpytela** [comment]
LGTM

**Pull request merged**